src/Pages/ChangePassword.cshtml

```diff
@@ -16,8 +16,28 @@
                 <button class="btn btn-outline-light">I don't feel Christmas!<i class="ml-1 fas fa-snowplow"></i></button>
             </form>
         }
-        <h3 class="text-light">Change Password</h3>
-        <p class="text-light">User: @User.FindFirst(ClaimTypes.Email).Value</p>
+        <h5 class="text-light mb-3 mt-3">User: <span class="text-warning">@User.FindFirst(ClaimTypes.Email).Value</span></h5>
+        <hr/>
+        <h5 class="text-light">Currency</h5>
+        <form method="post">
+            <div class="row justify-content-center">
+                <select class="mb-3" asp-for="Currency">
+                    <option hidden disabled selected value="">-Select-</option>
+                    <option value="1">EUR</option>
+                    <option value="2">USD</option>
+                    <option value="3">GBP</option>
+                    <option value="4">RUB</option>
```

Comment:
Could you make it possible for these values to be added from the cshtml.cs file itself? Like in the OnGet method we could set some values to a List and those would be displayed?

Reply:
Yes, this is placeholder, and it is possible to do `foreach` which generates these "options" for each item in the thing we are iterating. (as I understand in this case List)

Reply:
The thing is I'd like to start to try and implement the API, so I was wondering maybe you could do that foreach, just hardcode the list in the cshtml.cs, as in:

       var list = new List { "EUR", "USD", "GBP", "RUB"};
       foreach(var currency in list)
       { ... //set options ...}

Basically I want to have the base for me to easier implement the API :D

Reply:
oh ok, will be done soon, maybe even today :D